src/Adapter/Filesystem/SatelliteBuilder.php

```diff
@@ -162,6 +162,21 @@ public function build(): Configurator\SatelliteInterface
 
         $satellite->dependsOn(...$this->composerRequire);
 
+        $this->clearPipelines();
+
         return $satellite;
     }
+
+    private function clearPipelines(): void
+    {
+        $iterator = new \AppendIterator();
+
+        $iterator->append(new \GlobIterator($this->workdir.'/pipeline*.php', \FilesystemIterator::CURRENT_AS_FILEINFO | \FilesystemIterator::SKIP_DOTS));
+
+        foreach ($iterator as $file) {
+            if (is_file($file->getPathname())) {
```

Comment:
Est-ce que ici le `is_file` est nécessaire ? 
Sachant que juste avant dans ton itérator, tu récupère des fichier en `.php` ?

Reply:
Oui, ça évite un cas peu probable mais possible d'un dossier qui serait nommé `.php`